fetch-flow.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+# Fetches the latest binary release of Flow and writes the binaries to the `flow-bin/` directory.
+# The release name is read from the `FLOW_RELEASE` env variable, but it defaults to "dev".
+
+FLOW_RELEASE="${FLOW_RELEASE:-dev}"
```

Comment:
Is the `FLOW_RELEASE` a githash, which allows us to pin down some version of rust bins, maybe for trouble shooting?

Reply:
No, it's the name of a Github release. The default points to [this one](https://github.com/estuary/flow/releases/tag/dev). You can still use whatever Flow build you want when you're building locally, though. `fetch-flow.sh` just populates the `flow-bin/` directory in the repository root, but it's not run automatically, except in CI. So you can put whatever binaries you want in that directory if you're trying to debug against a specific flow build.